src/client/webapp/AppServiceWebAppAuthClient.ts

```diff
@@ -41,13 +42,16 @@ import {
 
 export class AppServiceWebAppAuthClient extends BaseAuthClient {
 
+    private cryptoUtils: CryptoUtils;
+
     /**
      * @param {AppSettings} appSettings
      * @param {Configuration} msalConfig
      * @constructor
      */
     constructor(appSettings: AppSettings, msalConfig: Configuration) {
         super(appSettings, msalConfig);
+        this.cryptoUtils = new CryptoUtils();
```

Comment:
```suggestion
        this.cryptoUtils = new CryptoUtils();
```

Can we instantiate this in `BaseAuthClient.ts` instead?